keyv/Plugin.h

```diff
@@ -0,0 +1,72 @@
+
+/* Copyright (c) 2017, Stefan.Eilemann@epfl.ch
+ *
+ * This file is part of Keyv <https://github.com/BlueBrain/Keyv>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 2.1 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#pragma once
+
+#include <keyv/types.h>
+#include <servus/uri.h>
+#include <lunchbox/compiler.h>
+
+namespace keyv
+{
+/** Interface for all Map plugins */
+class Plugin
+{
+public:
+    Plugin() {}
+    virtual ~Plugin() {}
+
+    /** @internal Needed by the PluginRegisterer. */
+    typedef Plugin InterfaceT;
+
+    /** @internal Needed by the PluginRegisterer. */
+    typedef servus::URI InitDataT;
+
+    /**
+     * Set the maximum number of non-commited writes.
```

Comment:
typo 'commited'. Besides, should be 'flushed' as there is a flush() function to explicitly commit all pending inserts (or fix flush() doxygen accordingly).